.drone.yml

```diff
@@ -254,6 +254,28 @@ pipeline:
     when:
       event: [ push, tag ]
 
+  gpg-sign:
+    image: plugins/gpgsign:1
+    pull: true
+    secrets: [ gpgsign_key, gpgsign_passphrase ]
+    detach_sign: true
+    files:
+      - dist/release/*-amd64
+      - dist/release/*-386
+      - dist/release/*-arm64
+      - dist/release/*-arm-5
+      - dist/release/*-arm-6
+      - dist/release/*-arm-7
+      - dist/release/*-mips
+      - dist/release/*-mips64
+      - dist/release/*-mipsle
+      - dist/release/*-mips64le
+      - dist/release/*-amd64.exe
+      - dist/release/*-386.exe
+      - dist/release/*.xz
+    when:
+      event: [ push, tag ]
```

Comment:
Do we really need the push event here?
Since release only happens on tagging, it should be sufficient here as well?